analytics/sensors/src/main/java/quarks/analytics/sensors/Filters.java

```diff
@@ -114,4 +114,27 @@ private Filters() {}
 
         return stream.filter(new Deadband<>(value, inBand));
     }
+    
+    /**
+     * A filter that discards tuples received during the "deadtime period"
+     * following a tuple this is allowed to pass through.
+     * <p>
+     * E.g., for a deadtime period of 30 minutes, after letting a tuple
+     * pass through, any tuples received during the next 30 minutes are
+     * filtered out.  Then the next arriving tuple is passed through and
+     * a new deadtime period is begun.
+     * </p><p>
+     * See {@link Deadtime} for information about changing the deadtime period
```

Comment:
Is this for the future, as there's no mechanism for the caller of this method to change the period?